Shared/sdk/net/bitstream.h

```diff
@@ -576,6 +576,10 @@ enum class eBitStreamVersion : unsigned short
     // 2024-09-04
     CheckDuplicateSerials,
 
+    // Add setElementOnFire function
```

Comment:
You must move this to the bottom of the list before merge.

Reply:
Done